packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -41,6 +42,12 @@ import { useIsSwitchingL2Chain } from './TransferPanelMainUtils'
 import { NonCanonicalTokensBridgeInfo } from '../../util/fastBridges'
 import { tokenRequiresApprovalOnL2 } from '../../util/L2ApprovalUtils'
 
+const onTxError = (error: any) => {
+  if (error.code !== 'ACTION_REJECTED') {
+    Sentry.captureException(error)
+  }
```

Comment:
Can we add a separate github issue for adding showing toast message in such cases + replace all the instances of `alert(` in codebase with Toast?

Reply:
there's an issue for window.alert #599